client/src/components/questionsAnswers/M-SubmissionModals.jsx

```diff
@@ -66,10 +64,11 @@ function SubModals(props) {
                 required
               />
             </div>
-            <div className="modalNickname">
+            <div className="qna-modalNickname">
               Nickname:
               <br />
               <input
```

Comment:
On my computer, your input fields have their left border cut off. I don't know if this is a consistent thing across browsers/computers/monitors, but I know it was something that I had to fix in my code. Here's what I did to fix it: https://stackoverflow.com/questions/44181487/text-input-border-gets-cut-off-in-form